packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -322,6 +322,18 @@ export async function sideload(manifestPath: string, command: commander.Command)
   }
 }
 
+export async function sideloadWeb(documentUrl: string, manifestPath: string, port: number, command: commander.Command) {
+  try {
+    const isTest: boolean = (command.test !== undefined);
+    const app: OfficeApp | undefined = command.app ? parseOfficeApp(command.app) : undefined;
+    await sideloadWebAddin(documentUrl, manifestPath, port, isTest);
+
+    // await sideloadAddIn(manifestPath, app, canPrompt);
```

Comment:
This feels incomplete.

Reply:
From my previous comments, I think we unify the sideload() function using the AppType so we don't need this.